spectre-meltdown-checker.sh

```diff
@@ -305,24 +367,24 @@ if [ "$opt_live" = 1 ]; then
 		opt_config=/boot/config-$(uname -r)
 	fi
 else
-	_echo "Checking for vulnerabilities against specified kernel"
+	_verbose "Checking for vulnerabilities against specified kernel"
```

Comment:
As per above, please keep this in standard output too, for context in copy/pastes